bigbluebutton-html5/imports/ui/components/nav-bar/settings-dropdown/component.jsx

```diff
@@ -97,8 +97,8 @@ class SettingsDropdown extends Component {
     let fullScreenIcon = 'fullscreen';
 
     if (isFullScreen) {
-      fullScreenLabel = intl.formatMessage(intlMessages.exitFullScreenLabel);
-      fullScreenDesc = intl.formatMessage(intlMessages.exitFullScreenDesc);
+      fullScreenLabel = intl.formatMessage(intlMessages.exitFullscreenLabel);
```

Comment:
Oh.. I followed 'ModalFullscreen' component name like other id such as userList and chatListItem.
I also changed 'app.userlist.*' to 'app.userList.*' and 'app.userlist.chatlistitem.*' to 'app.userList.chatListItem.*', because there component names are 'UserList' and 'ChatListItem'.
I am going to change fullScreenLabel to fullscreenLabel.